### ISSUE TITLE
KeyError: UROMAN_CMD

### ISSUE THREAD

**Issue by vikmary** [bltlab/paranames#2]
I am running the `recipes/paranames_pipeline.sh` script on the latest wikidata dump, and on stage 4 I get "KeyError: UROMAN_CMD" exception raise (see screenshot).

It seems like the script requires "UROMAN_CMD" environment variable to be set. What value should I set it to?
![image](https://user-images.githubusercontent.com/17520043/160064952-43ae9676-dcaa-43c7-8e9d-e5b98891f403.png)

**Comment by j0ma** [comment]
Hmm, seems like there were some old code leftovers from a previous version of the project in `paranames/util/script.py`. Uroman refers to the [Universal Romanizer](https://www.isi.edu/~ulf/uroman.html) project which we used to use for detecting some anomalies.

The unnecessary classes should now be gone as of [this commit](https://github.com/bltlab/paranames/commit/4719343f7b5b5729672e97c8bf667a176c5f63f7) and UROMAN_CMD should no longer be required. 

I'll leave the issue open for now, feel free to close it if the bug has been resolved on your end. 

And thanks again for testing out our project -- let us know if you run into further issues!
